v0/destinations/zendesk/transform.js

```diff
@@ -453,32 +477,20 @@ async function processIdentify(message, destinationConfig, headers) {
   );
   const url = endPoint + category.createOrUpdateUserEndpoint;
   const returnList = [];
+  let userId = await getUserId(message, headers);
+  if (!userId) {
+    userId = await getUserIdByExternalId(message, headers);
```

Comment:
Call this function only when `searchByExternalId` is enabled.

Reply:
yeah, updated